src/services/pool/decorators/pool.decorator.ts

```diff
@@ -69,6 +70,11 @@ export class PoolDecorator {
       poolService.setFeesSnapshot(poolSnapshot);
       poolService.setVolumeSnapshot(poolSnapshot);
       await poolService.setLinearPools();
+      if (sdkPool) {
+        pool.totalLiquidity = await sdkPools.liquidity(sdkPool);
+      } else {
+        poolService.setTotalLiquidity(prices, currency, tokens);
```

Comment:
I'd prefer we don't use this calculation at all anymore, so if the SDK calculation fails we should fallback to the subgraph value. That might even be what is happening in the SDK function too.

Reply:
removed